src/ptvsd/wrapper.py

```diff
@@ -1745,6 +1732,10 @@ def on_setDebuggerProperty(self, request, args):
             jmc = int(args.get('JustMyCodeStepping', 0)) > 0
             self.debug_options['DEBUG_STDLIB'] = not jmc
 
+        pydevd_request = copy.deepcopy(request)
```

Comment:
Could we use `_forward_request_to_pydevd` here? Note that this wouldn't need to be an `async_handler` in this case.